packages/patrol_cli/lib/src/features/drive/test_runner.dart

```diff
@@ -101,30 +158,45 @@ class TestRunner extends Disposable {
 
     _running = true;
 
+    final targetRunResults = <TargetRunResult>[];
+
     final testRunsOnAllDevices = <Future<void>>[];
     for (final device in _devices.values) {
       Future<void> runTestsOnDevice() async {
         for (final target in _targets) {
+          final targetRuns = <TargetRunStatus>[];
+          targetRunResults.add(
+            TargetRunResult(target: target, device: device, runs: targetRuns),
+          );
```

Comment:
I know it's probably not that much of an issue here, but having `TargetRunResult` as an `Equatable` and then mutating its fields seems like a risky maneuver

Reply:
Why? I don't think it breaks `Equatable`'s contract.

Reply:
I think it's ok. Besides the fact that i do not like mutability at all, but whether it is Equatable or not doesn't change anything here.